docs/matrix_format_spec.md

```diff
@@ -0,0 +1,144 @@
+# Format
+
+The secondary analysis pipelines shall write expression matrices and associated
+tabular metadata using the zarr version 2 directory store format. The format is
+specified more precisely
+[here](https://zarr.readthedocs.io/en/stable/spec/v2.html), but in general zarr
+stores contain groups and arrays. Groups can contain other groups and arrays.
+Arrays are stored chunked and compressed. The file names and directory
+structure of the zarr store convey the group and chunk structure.
+
+# Groups and Arrays
+
+Expression values and metadata are stored in seven arrays. These arrays are
```

Comment:
Should this say that the `Expression values and metadata are stored in eleven arrays` instead of seven?

Reply:
Yep, thanks